src/main/resources/containers.json

```diff
@@ -294,7 +294,7 @@
   {
     "artifact_id": "arquillian-tomcat-managed-5.5",
     "download": {
-      "url": "http://archive.apache.org/dist/tomcat/tomcat-5/v${version.org.apache.tomcat}/bin/apache-tomcat-${version.org.apache.tomcat}.zip"
+      "url": "http://archive.apache.org/dist/tomcat/tomcat-5/v5.5.36/bin/apache-tomcat-5.5.36.zip"
```

Comment:
The reason I hard-coded it because if try to support it using I couldn't find out latest version of 5.5 using g:"org.apache.tomcat" AND a:"apache-tomcat" which is maintained it in atlassian [repository](https://maven.atlassian.com/content/repositories/atlassian-public/). So I am not sure how to make it dynamic in this case